src/main/java/org/jabref/gui/importer/fetcher/WebSearchPaneViewModel.java

```diff
@@ -14,11 +15,9 @@
 import org.jabref.gui.DialogService;
 import org.jabref.gui.StateManager;
 import org.jabref.gui.importer.ImportEntriesDialog;
+import org.jabref.gui.linkedfile.DownloadLinkedFileAction;
 import org.jabref.gui.util.BackgroundTask;
-import org.jabref.logic.importer.CompositeIdFetcher;
-import org.jabref.logic.importer.ParserResult;
-import org.jabref.logic.importer.SearchBasedFetcher;
-import org.jabref.logic.importer.WebFetchers;
+import org.jabref.logic.importer.*;
```

Comment:
Please use our style  guidelines which described in the devdocs: https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/intellij-13-code-style.html#step-3-set-up-jabrefs-code-style

Reply:
Followed ! sorry for missing this